src/app/components/WorkDetail/WorkDetail.jsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { Link } from 'react-router';
+import {
+  isEmpty as _isEmpty,
+} from 'underscore';
+import appConfig from '../../../../appConfig';
+
+const WorkDetail = (props) => {
+  if (!props.detail) {
+    return null;
+  }
+
+  const { item } = props.detail;
+  const ebookUrl = appConfig.ereader[process.env.APP_ENV];
+
+  const parseEntities = (entities) => {
+    return (
+      <ul>
+        {entities.map((entity, i) => (
+          <li key={i.toString()}>
+            <Link to={{ pathname: '/', query: { author: `${entity.name}` } }}>{entity.name}, {entity.role}</Link>
```

Comment:
It is hypothetically possible that an entity will have multiple roles per work (e.g. author, editor). I'm not sure if any of the test records have this, but I have seen it in Gutenberg/OCLC data

Reply:
I guess it depends on how role is being handled in the data model. Is it a string or an array or either? That may dictate how the output is parsed.